frontend/src/components/ControlPanel/ControlPanel.tsx

```diff
@@ -59,53 +60,52 @@ function ControlPanel({
 		MODEL_DEFENCES.includes(defence.id)
 	);
 
-	// only allow configuration in sandbox
-	const showConfigurations = currentLevel === LEVEL_NAMES.SANDBOX;
+	// only allow configuration in sandbox and level 3
+	const showConfigurations =
+		currentLevel === LEVEL_NAMES.SANDBOX ||
+		currentLevel === LEVEL_NAMES.LEVEL_3;
```

Comment:
this is a fairly major design change! Before we were allowing defences toggling on and off but no configuration of the config item values.

Also we're still hiding the `<ModelBox>` in level 3. A user might wonder why those are missing from level 3 when all the other defences are available.

though it's hard for me to oppose the change, given the defences have almost zero practical relevance to the level as it is currently set up.

Reply:
Ooh, I think that was a mistake, possibly a merge conflict gone wrong... I'll fix that now.